projects/aca-content/src/lib/components/sidenav/components/expand-menu.component.html

```diff
@@ -10,7 +10,6 @@
       mat-button
       class="action-button full-width"
     >
-      <adf-icon *ngIf="item.icon" [value]="item.icon"></adf-icon>
```

Comment:
why icon is removed from the menu component? is that by design to not have any icons?

Reply:
Icons are not required in the design according to the requirement